scripts/archiveAndFlushAccessTokens.js

```diff
@@ -0,0 +1,451 @@
+/*!
```

Comment:
Shall this script be removed if the decision is not to archive access tokens?

Reply:
Strange ...  I thought I used `git mv` to change the name of the script file.  This file shouldn't exist anymore.  Anyhow, I"ll remove it.